src/signals/incident-management/components/CheckboxList/index.js

```diff
@@ -136,7 +139,7 @@ const CheckboxList = ({
 
           <input
```

Comment:
This element is only required for a nested structure which the `CheckboxList` component expects. For a flat structure, you can omit rendering the hidden input. You'll only have to change the condition for `displayToggle`; it should check for `toggleLabel` and `groupName`. The latter should not be a required prop.

Reply:
gedaan